src/app/services/requests.service.ts

```diff
@@ -85,10 +84,16 @@ export class RequestsService {
    */
   delete(url) {
     // Post an Empty object to create submission.
-    let body = JSON.stringify({});
+    const body = JSON.stringify({});
```

Comment:
I think the body is not in use any more. Can you delete it?

Reply:
I agree

Reply:
Done